platform/core/src/DICOMSR/dataExchange.js

```diff
@@ -81,7 +86,7 @@ const generateReport = measurementData => {
  * @param {object} dataSource The dataSource that you wish to use to persist the data.
  * @return {object} The naturalized report
  */
-const storeMeasurements = async (measurementData, dataSource) => {
+const storeMeasurements = async (measurementData, dataSource, options = {}) => {
```

Comment:
This was previously `additionalFindingTypes`, so I hope this change is intentional